packages/flutter/example/integration_test/native_jni_utils_test.dart

```diff
@@ -0,0 +1,172 @@
+// ignore_for_file: depend_on_referenced_packages
+
+@TestOn('vm')
+
+import 'dart:io';
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:jni/jni.dart';
+import 'package:sentry_flutter/src/native/java/sentry_native_java.dart';
+
+import 'utils.dart';
+
+final _customObject = CustomObject();
```

Comment:
Most (if not all) other tests have their varsity in the main function scope. Any reason to do this outside?